src/lsst/cm/tools/db/sqlalch_interface.py

```diff
@@ -149,11 +137,14 @@ def launch_jobs(self, level: LevelEnum, db_id: DbId, max_running: int) -> list[D
         db_id_list: list[DbId] = []
         entry = self.get_entry(level, db_id)
         handler = entry.get_handler()
-        n_running = self._count_jobs_at_status(StatusEnum.running)
-        if n_running >= max_running:
-            return db_id_list
+        n_running = 0
+        # n_running = self._count_jobs_at_status(StatusEnum.running)
```

Comment:
Leftover on purpose, or forgotten?  (DM discourages anonymous blocks of commented out code; if this is "handy" and you'd like to keep it perhaps just add a one-liner saying what it does/changes.)

Reply:
Added comment explaining that the commented code was what we actually wanted, but the function being called wasn't always working correctly, so I commented it out for now.